src/services/db/CommitServiceGithub.ts

```diff
@@ -0,0 +1,77 @@
+import { AxiosCacheInstance } from "axios-cache-interceptor"
+
+import UserWithSiteSessionData from "@root/classes/UserWithSiteSessionData"
+import { isReduceBuildTimesWhitelistedRepo } from "@root/utils/growthbook-utils"
+
+import { isFileAsset } from "./CommitServiceGitFile"
+import GitFileSystemService from "./GitFileSystemService"
+import GitHubService, { STAGING_LITE_BRANCH } from "./GitHubService"
+
+/**
+ * Responsibilities of this class
+ * 1. Creates all commits to staging
+ * 2. Creates non-asset related commits to staging-lite
+ */
+export default class CommitServiceGitHub extends GitHubService {
```

Comment:
btw for naming prefer to have service, handler, router etc as the suffix
 
ie. GitHubCommitService

Reply:
i realised this after I pushed out the other prs :sadge: 
not sure how well git handles rebasing for renames, is it ok if I did this in the upsteam pr?

Reply:
sure! upstream pr will merge into this PR right?

Reply:
nope, am intending to merge this into mainline first then upstream pr will merge into main line to keep the lines per merge small

Reply:
but ye this name change will go in by today